contribs/pam_slurm_adopt/pam_slurm_adopt.c

```diff
@@ -620,14 +625,17 @@ static void _log_init(log_level_t level)
 	log_init(PAM_MODULE_NAME, logopts, LOG_AUTHPRIV, NULL);
 }
 
-static int _load_cgroup_config()
-{
-    if (xcgroup_get_slurm_cgroup_conf() != SLURM_SUCCESS) {
-		info("read_slurm_cgroup_conf failed");
-		return SLURM_ERROR;
-	}
-	return SLURM_SUCCESS;
-}
+/* static int _load_cgroup_config() */
+/* { */
+/*         slurm_cgroup_conf = xmalloc(sizeof(slurm_cgroup_conf_t)); */
+/*         memset(slurm_cgroup_conf, 0, sizeof(slurm_cgroup_conf_t)); */
+/*         if (read_slurm_cgroup_conf(slurm_cgroup_conf) != SLURM_SUCCESS) { */
+/*                 info("read_slurm_cgroup_conf failed"); */
+/*                 return SLURM_ERROR; */
+/*         } */
+/*         return SLURM_SUCCESS; */
+/* } */
+
```

Comment:
What is this about?

Reply:
this is from the patch from schedmd. i just copied it verbatim (if that is the code they will use, this is it)

Reply:
OK!